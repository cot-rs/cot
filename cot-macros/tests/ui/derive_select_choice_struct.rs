use cot_macros::SelectChoice;

#[derive(SelectChoice)]
struct NotAnEnum {
    x: u8,
    y: u8,
}

fn main() {}
