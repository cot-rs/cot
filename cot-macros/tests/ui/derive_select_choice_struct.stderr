error: `SelectChoice` can only be derived for enums
 --> tests/ui/derive_select_choice_struct.rs:3:10
  |
3 | #[derive(SelectChoice)]
  |          ^^^^^^^^^^^^
  |
  = note: this error originates in the derive macro `SelectChoice` (in Nightly builds, run with -Z macro-backtrace for more info)
